pyproject.toml

```diff
@@ -1,3 +1,58 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "django-stubs"
+version = "5.2.0"
+description = "Mypy stubs for Django"
+readme = "README.md"
+license = "MIT"
+license-files = ["LICENSE.md"]
+requires-python = ">=3.10"
+authors = [
+    { name = "Maksim Kurnikov", email = "maxim.kurnikov@gmail.com" },
+]
+maintainers = [
+    { name = "Marti Raudsepp", email = "marti@juffo.org" },
+]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Typing :: Typed",
+    "Framework :: Django",
+    "Framework :: Django :: 5.0",
+    "Framework :: Django :: 5.1",
+    "Framework :: Django :: 5.2",
+]
+dependencies = [
+    "django",
+    "asgiref",
+    "django-stubs-ext>=5.2.0",
+    "tomli; python_version < '3.11'",
+    # Types:
+    "typing-extensions>=4.11.0",
+    "types-PyYAML",
+]
+
+[project.optional-dependencies]
+compatible-mypy = ["mypy>=1.13,<1.16"]
+oracle = ["oracledb"]
+redis = ["redis"]
+
+[project.urls]
+Homepage = "https://github.com/typeddjango/django-stubs"
+Funding = "https://github.com/sponsors/typeddjango"
+"Release notes" = "https://github.com/typeddjango/django-stubs/releases"
+
+[tool.hatch.build]
```

Comment:
Nit: Sort tool block alphabetically, i.e. move after `[tool.codespell]`?

Reply:
I kinda want to keep it close to the rest of the package metadata since it's use to build. No strong feeling tho